src/lwc/geFormRenderer/elevateCaptureGroup.js

```diff
@@ -0,0 +1,43 @@
+import apexAddToCaptureGroup from '@salesforce/apex/GE_GiftEntryController.addToCaptureGroup';
+import apexCreateCaptureGroup from '@salesforce/apex/GE_GiftEntryController.createCaptureGroup';
+import ElevateAuthorizedGift from './elevateAuthorizedGift';
+
+class ElevateCaptureGroup {
+
+    constructor(elevateBatchId) {
+        this.elevateBatchId = elevateBatchId;
+        this._hasAddRun = false;
+    }
+
+    async add(tokenizedGift) {
+        if (!this.elevateBatchId) {
+            this.elevateBatchId = await this.create();   
+        }
+
+        try {
+            let response = JSON.parse(await apexAddToCaptureGroup(tokenizedGift, this.elevateBatchId)).body;
+            let authorizedGift = new ElevateAuthorizedGift();
+            Object.keys(authorizedGift).forEach(
```

Comment:
Second the above, preferably with the response being the only thing we pass into `ElevateAuthorizedGift`.

Reply:
Ideally, this would be:
`
const authorizedGift = await apexAddToCaptureGroup(tokenizedGift, this.elevateBatchId);`

calling add should result in an instance of AuthorizedGift.  The AuthorizedGift instance should be instantiated inside of the add method and returned.

Reply:
Thanks for the feedback here all! I'm going to merge this PR into the parent branch in a second and then I'll make sure to address this there.